uni/ulsp/database.icn

```diff
@@ -96,14 +96,15 @@ class LSPDB(
    end
 
    method build_package_db()
-      local dirPath, db_paths
+      local dirPath, db_paths := [], dir
 
       dirPath := unicon_dir
 
-      db_paths := ipaths_get()
-
-      pop(db_paths) # remove the current directory
-
+      write("Looking in:")
```

Comment:
Should there be some extra writes that print out each directory that is added?

Reply:
We print it down below, but there was a duplicate "Looking in" that I took out now.